src/components/EventInfo/EventInfo.jsx

```diff
@@ -0,0 +1,36 @@
+import React from "react"
+
+import EventInfoStyles from "./EventInfoStyles"
+import Event49Img from "../../assets/images/event/McHacks-49.jpg"
+import Container from "./Container"
+import Image from "./Image"
+import Text from "./Text"
+
+const EventInfo = () => {
```

Comment:
Indent should be spaces 2

Reply:
Whoops! Apologies code editor was having some trouble, I'll correct this.